frontend/.eslintrc.js

```diff
@@ -28,6 +28,15 @@ module.exports = {
         "padded-blocks": "off",
         "quote-props": "off",
         "no-magic-numbers": "off",
+        "max-lines-per-function": "off",
+        "default-case": "off",
+        "no-console": "off",
+        "func-style": "off",
+        "space-before-function-paren": ["error", {
```

Comment:
```suggestion
        "space-before-function-paren": ["warn", {
```

As a style lint, this should only `warn`, right?